src/main/java/hudson/remoting/Launcher.java

```diff
@@ -270,12 +270,31 @@ public void setNoCertificateCheck(boolean ignored) throws NoSuchAlgorithmExcepti
             depends = "-workDir")
     @Nonnull
     public boolean failIfWorkDirIsMissing = WorkDirManager.DEFAULT_FAIL_IF_WORKDIR_IS_MISSING;
+    
+    /**
+     * Shows help message and then exits
+     * @since 3.36
+     */
+    @Option(name="-help",usage="Show this help message")
+    public boolean showHelp = false;
+
+    /**
+     * Shows version information and then exits
+     * @since 3.36
+     */
+    @Option(name="-version",usage="Shows the version of the remoting jar and then exits")
+    public boolean showVersion = false;
+
 
     public static void main(String... args) throws Exception {
         Launcher launcher = new Launcher();
         CmdLineParser parser = new CmdLineParser(launcher);
         try {
             parser.parseArgument(args);
+            if (launcher.showHelp && !launcher.showVersion) {
```

Comment:
If we specify version and help it shows the version. Should this be the behaviour? Just thinking out loud.

Reply:
Maybe include the version in the help output? I recall seeing some tools behave like that, but at the moment I'm not finding any examples. 

At a minimum, maybe help should take precedence over version.

Reply:
I was modeling this behavior after what jenkins.war does. If you run `java -jar jenkins.war --help --version` it prints the version. We can definitely switch the priority.

Reply:
I'm okay with being consistent with jenkins.war.